scripts/checkpoint_to_huggingface.py

```diff
@@ -21,7 +21,7 @@
 def push_to_huggingface(
     checkpoint_dir_paths: list[str],
     val_best: bool = True,
-    wandb_ids: Optional[list[str]] = None,
+    wandb_ids: list[str] = [],
```

Comment:
Out of curiosity: why not Optional anymore?

Reply:
That was a mistake, added it back in, thanks! Also updated the type hint slightly to the more up to date python way so we don't need to use `Optional`